db/model/model.js

```diff
@@ -8,27 +8,40 @@ exports.fetchTopics = () => {
 };
 
 exports.fetchArcticlesById = (id) => {
-  return (
-    db
-      .query(
-        `SELECT articles.*, (SELECT COUNT(*)::INT
+  return db
+    .query(
+      `SELECT articles.*, (SELECT COUNT(*)::INT
             FROM comments WHERE article_id=$1) AS comment_count
             FROM articles
             LEFT JOIN comments ON articles.article_id=comments.article_id WHERE articles.article_id=$1`,
-        [id]
-      )
-      .then(({ rows: [rows] }) => {
-        console.log(rows)
-        if (rows === undefined) {
-          return Promise.reject({ status: 400, msg: "Id not found" });
-        }
-        return rows;
-      })
-  );
+      [id]
+    )
+    .then(({ rows: [rows] }) => {
+      if (rows === undefined) {
+        return Promise.reject({ status: 400, msg: "Id not found" });
+      }
+      return rows;
+    });
+};
+
+exports.fetchCommentsById = (id) => {
+  return db
+    .query(
+      `SELECT *
+      FROM comments
+      WHERE article_id=$1
+      ORDER BY created_at DESC`,
+      [id]
+    )
+    .then(( {rows} ) => {
+      if (rows.length === 0) {
```

Comment:
There will be some articles where there are no comments, but they are valid articles and should just receive an empty array of comments, rather than an error. We'll need to check if the article exists, and if it does and has no comments, we'll send through the empty comments array.